src/Components/Web/src/WebEventData.cs

```diff
@@ -3,57 +3,57 @@
 
 using System;
 using System.Text.Json;
-using System.Threading.Tasks;
 using Microsoft.AspNetCore.Components.Rendering;
-using Microsoft.JSInterop;
 
 namespace Microsoft.AspNetCore.Components.Web
```

Comment:
Should this be in a .Internal namespace?

Reply:
We don't have an `.Internal` namespace in this project. Since it's a pattern we've been trying to minimise, I'm not sure it's warranted here. This type only gives you a special way of parsing JSON - it doesn't have any other powers, so the back-compatibility burden is minimal.